google/cloud/storage/testbench/gcs_object.py

```diff
@@ -16,9 +16,11 @@
 
 
 import base64
+import crc32c
```

Comment:
> gsutil uses the compiled `crcmod` package, with the c extension that comes when installed via pip, to do these calculations. Is there any reason you used this `crc32c` module instead?

I searched for "python crc32c" and that one came up first.

> although that should be fine for use in this project, since we're installing it via pip rather than bundling it with the project code, right?

I believe it should be fine.

I am happy to change to `crcmod`, do not care which one we use.

Reply:
My last point sounds like a problem if we're going to want to iteratively calculate a crc32c over several chunks, say, in a large resumable upload.  If this module has a way of doing that, I'm all for using it -- it's *much* faster than crcmod.

Reply:
Ooooh that is neat, then. Using this one sgtm!

And that would also be nice, yes!  Would also be nice if we could add an issue for it to that repo...